apps/web/lib/features/team-members-kanban-view.tsx

```diff
@@ -235,20 +230,12 @@ export const KanbanView = ({ kanbanBoardTasks }: { kanbanBoardTasks: IKanban })
 									) : null}
 									<>{provided.placeholder}</>
 								</div>
-								<Container fullWidth={fullWidth} className={clsxm('w-full !mx-0 px-8')}>
```

Comment:
@Anishali2 where this "footer" go now? I can't see it in PR, but I see it visible in the video you recorded, so how we display it if you removed that code here?

Reply:
It's in the main-layout I recently removed added the class footer `footerClassName` and from main layout of kanban screen and so it's visible in the main app screen so we don't need it anymore

Reply:
I've done this before for scrolling and now the requirements are adding scroll on the full screen so we actually don't need it anymore